src/main/java/bio/terra/service/configuration/ConfigurationService.java

```diff
@@ -113,6 +113,17 @@ public ConfigListModel setConfig(ConfigGroupModel groupModel) {
     return new ConfigListModel().items(priorConfigList).total(priorConfigList.size());
   }
 
+  public void setConfigParameterValue(ConfigEnum config, String newValue, String label) {
+    bio.terra.model.ConfigModel retryConfigModel =
+        new bio.terra.model.ConfigModel()
+            .name(config.name())
+            .configType(bio.terra.model.ConfigModel.ConfigTypeEnum.PARAMETER)
+            .parameter(new bio.terra.model.ConfigParameterModel().value(newValue));
+    ConfigGroupModel group =
+        new bio.terra.model.ConfigGroupModel().label(label).addGroupItem(retryConfigModel);
+    setConfig(group);
```

Comment:
Don't need FQNs here, the IDE doesn't work well with autogenerated classes unfortunately
```suggestion
    ConfigModel retryConfigModel =
         new ConfigModel()
            .name(config.name())
            .configType(ConfigModel.ConfigTypeEnum.PARAMETER)
            .parameter(new ConfigParameterModel().value(newValue));
    setConfig(new ConfigGroupModel().label(label).addGroupItem(retryConfigModel));
```